### PR TITLE
Task3 Update calculator

### PR DESCRIPTION
**Pull request by IrynaSemenenko**
Update calculator by using "switch"

### CODE REVIEW

File: Calculator2/src/Calculator2.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+public class Calculator2 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+
+        System.out.println("Enter the first number");
+        if (sc.hasNextDouble()) {
+            double firstValue = sc.nextDouble();
+            System.out.println("Input is correct");
+
```
Comment (Hmeliumm) [inline Calculator2/src/Calculator2.java:11]: Please remove extra line

File: Calculator2/src/Calculator2.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+            System.out.println("Enter an operation (for example: +, -, *, /, %)");
+            String operation = sc.next();
+            switch (operation) {
+                case "+", "-", "*", "/", "%" -> System.out.println("User inputs supported operations");
```
Comment (Hmeliumm) [inline Calculator2/src/Calculator2.java:16]: ahah, looks interesting) Please avoid using switch case for such cases

File: Calculator2/src/Calculator2.java [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+                    case "+" -> {
+                        double result1 = firstValue + secondValue;
+                        System.out.println("Result is: " + result1);
+                    }
+                    case "-" -> {
+                        double result2 = firstValue - secondValue;
+                        System.out.println("Result is: " + result2);
+                    }
+                    case "/" -> {
+                        if (firstValue == 0 | secondValue == 0) {
```
Comment (Hmeliumm) [inline Calculator2/src/Calculator2.java:35]: if first value -  it's ok

File: Calculator2/src/Calculator2.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            }
+
+            System.out.println("Enter the second number");
+            if (sc.hasNextDouble()) {
+                double secondValue = sc.nextDouble();
+                System.out.println("Input is correct");
+
+                switch (operation) {
+                    case "+" -> {
+                        double result1 = firstValue + secondValue;
```
Comment (Hmeliumm) [inline Calculator2/src/Calculator2.java:27]: Please use one variable for result

File: Calculator2/src/Calculator2.java [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+                        }
+                        double result3 = firstValue / secondValue;
+                        System.out.println("Result is: " + result3);
+                    }
+                    case "*" -> {
+                        double result4 = firstValue * secondValue;
+                        System.out.println("Result is: " + result4);
+                    }
+                    case "%" -> {
+                        if (firstValue == 0 | secondValue == 0) {
```
Comment (Hmeliumm) [inline Calculator2/src/Calculator2.java:46]: It's ok, as we get in result 0

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Calculator2/src/Calculator2.java (hunk 1 above)
Comment (IrynaSemenenko) [reply Calculator2/src/Calculator2.java]: Done

File: Calculator2/src/Calculator2.java (hunk 2 above)
Comment (IrynaSemenenko) [reply Calculator2/src/Calculator2.java]: Done. I have changed 'switch' to 'if'

File: Calculator2/src/Calculator2.java (hunk 3 above)
Comment (IrynaSemenenko) [reply Calculator2/src/Calculator2.java]: Done. I have removed 'firstValue == 0'

File: Calculator2/src/Calculator2.java (hunk 4 above)
Comment (IrynaSemenenko) [reply Calculator2/src/Calculator2.java]: Done

File: Calculator2/src/Calculator2.java (hunk 5 above)
Comment (IrynaSemenenko) [reply Calculator2/src/Calculator2.java]: Done. I removed 'if' in this case.

**Review by Hmeliumm — APPROVED** [review]

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**